modules/gql.yml

```
@@ -0,0 +1,17 @@
+name: gql
+description: ⚡️ Minimal GraphQL Client + Code Generation for Nuxt3. Auto imports GraphQL Operations for easy execution.
```

Marketing: Please avoid Nuxt version in module description.

```suggestion
description: Minimal GraphQL Client + Code Generation for Nuxt. Auto imports GraphQL Operations for easy execution.
```